lib/tasks/contributor_stats.rake

```diff
@@ -0,0 +1,10 @@
+namespace :fromthepage do
+  desc "transcription stats from the previous 24 hours"
+  task :contributor_stats, [:collection_id] => :environment do |t,args|
```

Comment:
How hard would it be for the rake task to take one (or more) email addresses and use them for the "to" in the mailer?  That's most flexible.